app/views/users/show.html.haml

```diff
@@ -6,6 +6,11 @@
       %h1.user_profile_header= @user.name
       %p= link_to 'Edit profile', users_edit_profile_path
 
+      - if @user.about_me?
+        %p= @user.about_me
```

Comment:
What the hell!! I only wrote the first line of that comment, no idea where that other text has come from!!

Think it may have been one of sams comments on my code, no idea how it ended up in that last comment though :S